src/asymmetric_cipher.c

```diff
@@ -293,43 +300,41 @@ static int p11prov_rsaenc_decrypt(void *ctx,
 endsess:
     ret = f->C_CloseSession(session);
     if (ret != CKR_OK) {
-        P11PROV_raise(encctx->provctx, ret,
-                      "Failed to close session %lu", session);
+        P11PROV_raise(encctx->provctx,
+                      ret,
+                      "Failed to close session %lu",
+                      session);
     }
 
     return result;
-
 }
 
 static struct {
     CK_MECHANISM_TYPE type;
     unsigned int ossl_id;
     const char *string;
 } padding_map[] = {
-    { CKM_RSA_X_509, RSA_NO_PADDING, OSSL_PKEY_RSA_PAD_MODE_NONE },
-    { CKM_RSA_PKCS, RSA_PKCS1_PADDING, OSSL_PKEY_RSA_PAD_MODE_PKCSV15 },
-    { CKM_RSA_PKCS_OAEP, RSA_PKCS1_OAEP_PADDING, OSSL_PKEY_RSA_PAD_MODE_OAEP },
-    { CKM_RSA_X9_31, RSA_X931_PADDING, OSSL_PKEY_RSA_PAD_MODE_X931 },
-    { CK_UNAVAILABLE_INFORMATION, 0, NULL }
-};
+    {CKM_RSA_X_509, RSA_NO_PADDING, OSSL_PKEY_RSA_PAD_MODE_NONE},
+    {CKM_RSA_PKCS, RSA_PKCS1_PADDING, OSSL_PKEY_RSA_PAD_MODE_PKCSV15},
+    {CKM_RSA_PKCS_OAEP, RSA_PKCS1_OAEP_PADDING, OSSL_PKEY_RSA_PAD_MODE_OAEP},
+    {CKM_RSA_X9_31, RSA_X931_PADDING, OSSL_PKEY_RSA_PAD_MODE_X931},
+    {CK_UNAVAILABLE_INFORMATION, 0, NULL}};
 
 /* only the ones we can support */
 static struct {
     const char *name;
     CK_MECHANISM_TYPE digest;
     CK_RSA_PKCS_MGF_TYPE mgf;
-} digest_map[] = {
-    { "SHA3-512", CKM_SHA3_512, CKG_MGF1_SHA3_512 },
-    { "SHA3-384", CKM_SHA3_384, CKG_MGF1_SHA3_384 },
-    { "SHA3-256", CKM_SHA3_256, CKG_MGF1_SHA3_256 },
-    { "SHA3-224", CKM_SHA3_224, CKG_MGF1_SHA3_224 },
-    { "SHA512", CKM_SHA512, CKG_MGF1_SHA512 },
-    { "SHA384", CKM_SHA384, CKG_MGF1_SHA384 },
-    { "SHA256", CKM_SHA256, CKG_MGF1_SHA256 },
-    { "SHA224", CKM_SHA224, CKG_MGF1_SHA224 },
-    { "SHA1", CKM_SHA_1, CKG_MGF1_SHA1 },
-    { NULL, 0, 0 }
-};
+} digest_map[] = {{"SHA3-512", CKM_SHA3_512, CKG_MGF1_SHA3_512},
```

Comment:
here going to a new line would be better, is it configurable?
Or does it do it only if the inititalization argument is too long like with the padding_map case?